adds entries for canceling and archiving runs to dashboard page (#2583)
cypress-io/cypress-documentation: source/guides/dashboard/runs.md

{% imgTag /img/dashboard/run-details.png "run-details" %}

# Cancelling runs

You can cancel a run currently in progress from the Dashboard. Runs can only be cancelled by members of a project.

To cancel a run, click on a run in progress from the run list, then click on "Cancel Run" in the upper-right corner of the run details page.

> Cancelling a run in progress cannot be undone. Once you click "Cancel Run", a confirmation message will appear. Click the red "Yes, cancel this run" button to proceed.

Reviewer: I might write this in bullet format to make the steps easier to read.

```
To cancel a run:

- Click on a run...
- Click **Cancel Run** in the upper right corner...
- Click **Yes, cancel this run** to confirm. **Note: this cannot be undone**
```

{% video local /img/snippets/cancelling-run.mp4 %}

Once a run is cancelled:

- The run status will update to cancelled
- All incompleted instances of the run will be marked as cancelled

> - All test runners associated with the run will exit with an error

Reviewer: I don't understand this point. I think this is meant to mean that any `cypress run` currently in progress associated with the run will exit with an error code, but I'm not sure this is actually how this behaves.

Author: That is correct, it does throw an error. I can add a screenshot of what that error output looks like.

> - A message will appear on the run details page showing the time of cancellation and the user that made the request

Reviewer: ```suggestion
- A message will appear on the run details page showing the time of cancellation and the user that cancelled the run
```

> - The run will update to cancelled in GitHub if integration is enabled

Reviewer: Also link to GitHub Integration here. 

```suggestion
- The run will display as **Cancelled** in the associated GitHub pull request if {% url "GitHub Integration" github-integration %} is enabled
```

> If you have Analytics enabled, cancelled runs will appear in the Runs over Time chart. Runs can also be filtered by cancellation status. 

Reviewer: ```suggestion
If you have Analytics enabled, cancelled runs will appear in the {% url "Runs over time" analytics#Runs-over-time %} chart. Runs can also be filtered by cancellation status. 
```

> Cancelled runs do not count against your test recording limit.

Reviewer: Oh, is this true? Even if 90% of the specs and tests completed running, and we only cancelled when the last spec was running, *none* of the test recordings count in their billing?

Reviewer 2: That's not accurate, no. We count all tests recorded before the run was canceled. If they ran 90% of their test suite, then canceled, that first 90% would still be included in their usage.

Author: Yes, I meant the tests that hadn't run yet. I'll definitely clarify that.

Author: I also confirmed that when specs have already run and recorded, the test results are available in the run details. I'll add that info as well.

# Archiving runs

> Runs that have been cancelled or are in an errored state can be archived from the Dashboard. Archiving a run will remove it from the run list and Analytics reporting.

Reviewer: ```suggestion
Runs that have been cancelled or are in an errored state can be archived from the Dashboard. Archiving a run will remove it from the run list and {% url "Analytics" analytics %} reporting.
```

Archived runs can still be accessed by the URL to that run. The format is:

> `dashboard.cypress.io/projects/{project ID}/runs/{run number}`

Reviewer: ```suggestion
`https://dashboard.cypress.io/projects/{project ID}/runs/{run number}`
```

Archived runs can be restored from the run detail page.

{% imgTag /img/dashboard/restore-from-archive.png "restore-from-archive" %}

> 

Reviewer: Ok, we need to clarify how Archived runs count against their billing as well. People will definitely ask that. "Can I just archive all my runs and they will be removed from my test recording limit"?
